### PR TITLE
[42] Sermon Card

### PR DESCRIPTION
**Pull request by Brandt-Bowling**

### CODE REVIEW

File: frontend/src/pages/watch/sermons/index.js
Comment (thedomcab) [inline frontend/src/pages/watch/sermons/index.js:23]: Could you remove this statement?

**Review by thedomcab — DISMISSED** [review]

**Comment by Brandt-Bowling** [comment]
@thedomcab I'm not sure how to get the cards to not overlap just before a transition between sizes
Screenshot 2023-12-18 at 8 29 16 PM

There's also a bit more space between the end of the text and the bottom of the card compared to the figma design. I'm not sure how to fix that directly in the SermonCard component without editing the shared Card component which would affect the events card as well.

For reference:
figma
Screenshot 2023-12-18 at 8 32 15 PM

mine
Screenshot 2023-12-18 at 8 33 01 PM

File: frontend/src/components/page-watch/index/sermonCard.js
Comment (thedomcab) [inline frontend/src/components/page-watch/index/sermonCard.js:6]:
```suggestion
    <div className="pt-1 md:pt-2">
```

File: frontend/src/components/page-watch/index/sermonCard.js
Comment (thedomcab) [inline frontend/src/components/page-watch/index/sermonCard.js:7]:
```suggestion
    <div className="pt-1 md:pt-2">
      <p className="leading-6 font-normal text-lg mb-0">
```

File: frontend/src/components/page-watch/index/sermonCard.js
Comment (thedomcab) [inline frontend/src/components/page-watch/index/sermonCard.js:23]:
```suggestion
    <Card img={img} date={formattedDate} title={title} containerCss="md:pb-[3.5625rem]">
```

File: frontend/src/components/page-watch/index/sermons.js
Comment (thedomcab) [inline frontend/src/components/page-watch/index/sermons.js:27]:
```suggestion
   <div className="grid grid-cols-2 lg:grid-cols-3 gap-x-5 gap-y-8 py-36 max-w-container px-4">
```

File: frontend/src/components/page-watch/index/sermons.js
Comment (thedomcab) [inline frontend/src/components/page-watch/index/sermons.js:30]:
```suggestion
          <SermonCard 
             key={i}
```

**Review by thedomcab — COMMENTED** [review]
I have made a PR for changes outside the files that needed to be changed to match figma and being more responsive.  notice alot of set wiidth height which caused the overlaying issue.  here the PR: https://github.com/hmcc-global/hmccaa-web/pull/105
you can pull the changes on this branch. or you can approve my changes and i can merge once approved and you can rebase with master. your choice.

**Review by thedomcab — COMMENTED** [review]
it seems you add my suggestions. is there a reason why?

**Comment by Brandt-Bowling** [comment]
> it seems you add my suggestions. is there a reason why?

I guess I got confused as to the process?

**Comment by thedomcab** [comment]
I did not pull your branch down to make changes. I  just did other components that need to be changed to make the items more responsive.  i realize that i did not catch some html elements had set witdth and height which I did not catch so i fixed those things and customize a component to feed in custom tailwind css for different htlml for the Card component.  I hope this clarification.

**Comment by Brandt-Bowling** [comment]
> I did not pull your branch down to make changes. I  just did other components that need to be changed to make the items more responsive.  i realize that i did not catch some html elements had set witdth and height which I did not catch so i fixed those things and customize a component to feed in custom tailwind css for different htlml for the Card component.  I hope this clarification.  

So what should I do next?

**Comment by thedomcab** [comment]
> > I did not pull your branch down to make changes. I  just did other components that need to be changed to make the items more responsive.  i realize that i did not catch some html elements had set witdth and height which I did not catch so i fixed those things and customize a component to feed in custom tailwind css for different htlml for the Card component.  I hope this clarification.
> 
> So what should I do next?

just commit my suggestions. okay?

File: frontend/src/components/page-watch/index/sermons.js
Comment (thedomcab) [inline frontend/src/components/page-watch/index/sermons.js:29]: remove the div here.

File: frontend/src/components/page-watch/index/sermons.js
Comment (thedomcab) [inline frontend/src/components/page-watch/index/sermons.js:44]: remove div here.  By removing the div will make the Sermon Card have the same height in each row.  i notice that the height are not consistent with the outer divs. after that all should be good.

**Review by thedomcab — APPROVED** [review]

**Pull request merged**
